README.md

```diff
@@ -290,6 +290,7 @@ By default, both `pip-compile` and `pip-sync` will look first
 for a `.pip-tools.toml` file and then in your `pyproject.toml`. You can
 also specify an alternate TOML configuration file with the `--config` option.
 
+It is possible to specify configuration values both globally and tool-specific.
```

Comment:
Let's avoid saying tool-specific since the document doesn't record what is a "tool" in different contexts and people may be getting different incorrect ideas.
"command" is probably fine. Could also be a sub-command, I suppose. Let's see if others have better ideas, though...

Reply:
Reworded to "command".